docs/development/managing-users/index.md

```diff
@@ -0,0 +1,43 @@
+# Managing users
+
+To manage users you will need an admin token and connection to the REANA-Server component:
+
+```console
+$ kubectl exec -ti reana-server-xxxx bash
+> read -s REANA_ADMIN_ACCESS_TOKEN
+>
+```
+
+## Create users
+
+```console
+> flask reana-users create -e john.doe@cern.ch
+User was successfully created.
+ID                                     EMAIL                 ACCESS_TOKEN
+aa37d63d-3186-45d5-aa40-5d221cb170bf   john.doe@google.com   xxxxxxxxxxxx
+```
+
+## List users
+
+```console
+> flask reana-users get
```

Comment:
`list` instead of `get`?  Former is usually returning many rows, latter is for returning one row properties.

Reply:
I think that for this command we started having the "show only one user user" use case, but things have changed. I will amend, thanks!